python/lsst/source/injection/bin/make_injection_pipeline.py

```diff
@@ -120,6 +120,32 @@ def build_argparser():
         help="Config override for a task, in the format 'label:key=value'.",
         action="append",
     )
+    parser.add_argument(
+        "-ip",
+        "--input-pipelines",
+        type=str,
+        help="""Location(s) of additional input pipeline definition YAML
+        file(s). Tasks from input_pipelines will be added to the injection
+        pipeline.""",
```

Comment:
As this is code, and not a docstring, you're allowed to go over the 79 character limit. It might be nicer to format this as:
```
        help="""Location(s) of additional input pipeline definition YAML file(s).
        Tasks from input_pipelines will be added to the injection pipeline.""",
```